src/test/java/org/libgit2/jagged/RepositoryTest.java

```diff
@@ -37,22 +37,12 @@ public void testOpenNonExistingRepositoryFails()
             @SuppressWarnings("unused")
             Repository failure = new Repository(new File(getTempDir(), "doesnotexist").getAbsolutePath());
         }
-        catch (GitException e)
+        catch (Throwable e)
```

Comment:
Don't remove the Windows and Unix error messages.  Add another one, with a test around Mingw.